src/main/resources/application.conf

```diff
@@ -0,0 +1,20 @@
+jmt {
```

Comment:
Does this replace the default values from `reference.conf` in JMT?

Reply:
Oops, this wasn't meant to be committed yet.